src/query/models/query_context.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
nit: 2019

Reply:
Not when I wrote it, I don't think :D. I'll update it.

Reply:
Yeah, weird situation since it got spun off into a new PR I guess?